configs/config.go

```diff
@@ -9,14 +9,15 @@ import (
 )
 
 type Config struct {
-	PostgresUrl      string `mapstructure:"POSTGRESQL_URL"`
-	HttpPort         string `mapstructure:"HTTP_PORT"`
-	GrpcPort         string `mapstructure:"GRPC_PORT"`
-	GraphqlPort      string `mapstructure:"GRAPHQL_PORT"`
-	HealthcheckPort  string `mapstructure:"HEALTHCHECK_PORT"`
-	PostgresInitFile string `mapstructure:"POSTGRES_INIT"`
-	PrivateKeyFile   string `mapstructure:"PRIVATE_KEY_FILE"`
-	RedisAddress     string `mapstructure:"REDIS_ADDRESS"`
+	PostgresUrl             string `mapstructure:"POSTGRESQL_URL"`
+	HttpPort                string `mapstructure:"HTTP_PORT"`
+	GrpcPort                string `mapstructure:"GRPC_PORT"`
+	GraphqlPort             string `mapstructure:"GRAPHQL_PORT"`
+	HealthcheckPort         string `mapstructure:"HEALTHCHECK_PORT"`
+	PostgresMigrationsPath  string `mapstructure:"POSTGRES_MIGRATIONS_PATH"`
+	PrivateKeyFile          string `mapstructure:"PRIVATE_KEY_FILE"`
+	RedisAddress            string `mapstructure:"REDIS_ADDRESS"`
+	PostgresDatabaseVersion uint   `mapstructure:"POSTGRES_DATABASE_VERSION"`
```

Comment:
one part of migration config is called `PostgresMigrationsPath` and `POSTGRES_MIGRATIONS_PATH`, another - `PostgresDatabaseVersion` and `POSTGRES_DATABASE_VERSION` ... please align it